packages/esm-patient-chart-app/src/banner/contact-details/contact-details.component.tsx

```diff
@@ -87,6 +92,66 @@ const Relationships: React.FC<{ patientId: string }> = ({ patientId }) => {
   );
 };
 
+const PatientLists: React.FC<{ patientId: string }> = ({ patientId }) => {
+  const { t } = useTranslation();
+  const { data: formattedPatientLists, isLoading } = usePatientLists(patientId);
+  const [showPatientListDetails, setShowPatientListDetails] = React.useState(false);
+  const togglePatientListDetails = React.useCallback((event: MouseEvent) => {
+    event.stopPropagation();
+    setShowPatientListDetails((value) => !value);
+  }, []);
+
+  return (
+    <>
+      <p className={styles.heading}>
+        {t('patientLists', 'Patient Lists')}({formattedPatientLists?.length})
+        <Button
+          kind="ghost"
+          renderIcon={(props) =>
+            showPatientListDetails ? <ChevronUp size={16} {...props} /> : <ChevronDown size={16} {...props} />
+          }
+          iconDescription="Toggle patient List Details"
+          onClick={togglePatientListDetails}
+          style={{ marginTop: '-0.25rem' }}
+        >
+          {showPatientListDetails ? t('seeLess', 'See less') : t('seeAll', 'See All')}
```

Comment:
`See less` and `See All` Please confirm from the designs that should the starting letter of second word should be capital or lower case.
Thanks!